src/Components/Scheduling/SchedulingColumn.tsx

```diff
@@ -118,14 +118,14 @@ const placeInlineBlocks = (layered_data: APICourseBlock[][], filter: any): JSX.E
         top: `${start_time_to_top(layer[0].start_time)}%`
       }}>
         { layer.map((block: APICourseBlock) => (
-          < SchedulingBlock course_instance={block} visible={filter[block.course_number]} linkIDs={block.scheduled} inline={true} key={`deep-unravel-block-${JSON.stringify(block)}`}/>
+          < SchedulingBlock course_instance={block} visible={filter.get(block.course_number)!} linkIDs={block.scheduled} inline={true} key={`deep-unravel-block-${JSON.stringify(block)}`}/>
```

Comment:
It is a force unwrap. I know that when the code reaches this point that that value must be in the filter (since if it wasn't the block wouldn't be there in the first place). If it turns out to be undefined it will throw an error but that isn't possible from there.